go/chunks/chunk.go

```diff
@@ -2,6 +2,7 @@
 // Licensed under the Apache License, version 2.0:
 // http://www.apache.org/licenses/LICENSE-2.0
 
+// Package chunks contains the core storage abstractions and implementations used in Noms.
```

Comment:
Trying to be a little more specific:

Package chunks provides facilities for representing, storing, and fetching content-addressed chunks of Noms data.